universal-login-react/src/ui/TopUp/Fiat/TopUpWithFiat.tsx

```diff
@@ -54,7 +56,7 @@ export const TopUpWithFiat = ({walletService, modalClassName, logoColor}: TopUpW
       return <Ramp
         address={contractAddress}
         amount={stringToEther(amount)}
```

Comment:
I suppose that If we support dai amount we can't just do stringToEther here

Reply:
Yes, the name can be confusing... I renamed it to `stringToWei`. Should be more descriptive now 😄